CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Host] In host and advanced view, display MDADM health information (PR [#8190](https://github.com/vatesfr/xen-orchestra/pull/8190))
+
```

Comment:
```suggestion
- [Host/Advanced] In host's advanced tab, display MDADM health information  (PR [#8190](https://github.com/vatesfr/xen-orchestra/pull/8190))

```

Reply:
This information will be displayed in the advanced tab and in the host's view